scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -16,7 +16,7 @@ namespace <%= namespace %>.SDK {
         }
 
         public enum ErrorType {
-            HTTP, GraphQL, UserError, NativePaymentProcessingError
+            HTTP, GraphQL, UserError, NativePaymentProcessingError, WebCheckoutVerificationError
```

Comment:
@dengjeffrey Is NativePaymentProcessingError something we use for Apple Pay? I think the errors are different in that one is related to Apple Pay processing and the other is related to validating a checkout. I might be better from a game dev perspective to just use one of these though.